### PR TITLE
Add runtime test for multiple `--type` in `cilium monitor`

### PR DESCRIPTION
**Pull request by nebril**
Had this test in a superseded PR, salvaged it so we check if this option actually work.

### CODE REVIEW

**Comment by nebril** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

File: test/runtime/monitor.go [hunk 1]
```diff
@@ -123,6 +123,29 @@ var _ = Describe("RuntimeValidatedMonitorTest", func() {
 			Expect(res.Output().String()).Should(ContainSubstring(v))
 			vm.SampleContainersActions(helpers.Delete, helpers.CiliumDockerNetwork)
 		}
+
+		By(fmt.Sprintf("all types together"))
```
Comment (eloycoto) [inline test/runtime/monitor.go]: I would remove `fmt.Sprintf` here

File: test/runtime/monitor.go [hunk 2]
```diff
@@ -123,6 +123,29 @@ var _ = Describe("RuntimeValidatedMonitorTest", func() {
...
+		vm.ContainerExec(helpers.App1, helpers.Ping(helpers.Httpd1))
+		helpers.Sleep(10)
+		cancel()
+
+		Expect(res.CountLines()).Should(BeNumerically(">", 3))
+		output := res.Output().String()
+		for _, v := range eventTypes {
+			Expect(output).Should(ContainSubstring(v))
+		}
+		vm.SampleContainersActions(helpers.Delete, helpers.CiliumDockerNetwork)
```
Comment (eloycoto) [inline test/runtime/monitor.go]: You don't need to do this here. In fact is a bad practice, all the cleanups should happen after `vm.ReportFailed`

**Review by eloycoto — CHANGES_REQUESTED** [review]

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
@eloycoto thanks, fixed

**Comment by eloycoto** [comment]
test-me-please

**Review by eloycoto — APPROVED** [review]

**Pull request merged**